### PR TITLE
Furniture.xml Style (See #789)

### PR DESCRIPTION
**Pull request by aleserena**
standarized indentation and style for Furniture.xml

### CODE REVIEW

**Comment by TomMalbran** [comment]
You are using Tabs instead of 4 spaces for the indentation.

**Comment by vogonistic** [comment]
When you've fixed the conflicts and addressed the comments, remove `[conflict]` from the title. Thanks!

File: Assets/StreamingAssets/Data/Furniture.xml [hunk 1]
```diff
@@ -27,7 +26,7 @@
           <!-- 1 means a regular amout is exchanged -->
           <!-- Walls will likely have low values, whilst tables and so on will have values close to 1 (if not 1). -->
           <!-- Leave default value (i.e. nothing) if the furniture does not have any isolation effect. -->
-          <Param name="thermal_diffusivity" value="0.0" />
+            <Param name="thermal_diffusivity" value="0.0" />
```
Comment (NogginBops) [inline Assets/StreamingAssets/Data/Furniture.xml]: Why is this farther in?

File: Assets/StreamingAssets/Data/Furniture.xml [hunk 2]
```diff
@@ -115,13 +112,13 @@
         <MovementCost>1</MovementCost>
         <LinksToNeighbours>true</LinksToNeighbours>
 
-        <BuildingJob jobTime="0">
-        </BuildingJob>
+        <BuildingJob jobTime="0" />
```
Comment (NogginBops) [inline Assets/StreamingAssets/Data/Furniture.xml]: Does the code handle this correctly?

File: Assets/StreamingAssets/Data/Furniture.xml (hunk 1 above)
Comment (aleserena) [inline Assets/StreamingAssets/Data/Furniture.xml]: actually I forgot to update the commented lines.

**Comment by aleserena** [comment]
See #789

**Pull request closed without merging**